src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -38,6 +38,7 @@ export default opts => {
     artworkLoader: gravityLoader(id => `artwork/${id}`),
     artworksLoader: gravityLoader("artworks"),
     bidderLoader: gravityLoader(id => `bidder/${id}`),
+    exchangeRatesLoader: gravityLoader("exchange_rates"),
```

Comment:
This might benefit from being cached in Metaphysics for a while (since these only change daily). That can be done with invoking: `gravityLoader(path, {}, { requestThrottleMs: 1000 * 60 * 60 * 6 }` to cache for 6 hours, for instance.

Reply:
Good call, added.